### ISSUE TITLE
Documentation bug: event example code TypeErrors

### ISSUE THREAD

**Issue by lf-** [mk-fg/python-pulse-control#16]
```
~ » python3.6 patest.py
Traceback (most recent call last):
  File "patest.py", line 10, in <module>
    print('Event types:', ', '.join(p.event_types))
TypeError: sequence item 0: expected str instance, EnumValue found
```

File contents:

```python
import pulsectl


def event_cb(evt):
    print(evt)
    print(dir(evt))


with pulsectl.Pulse('test1') as p:
    print('Event types:', ', '.join(p.event_types))
    print('Event facilities:', ', '.join(p.event_facilities))
    print('Event masks:', ', '.join(p.event_masks))
```

**Comment by mk-fg** [comment]
Right, forgot to update these when changed strings to enum values.
Thanks for reporting, should be fixed in 517429f.

**Comment by mk-fg** [comment]
Or rather 79b5f9e, as that I'm not sure what that str.join is even doing there :)
